packages/docusaurus-theme-openapi/src/theme/ApiDemoPanel/VSCode/index.tsx

```diff
@@ -21,7 +21,8 @@ interface Props {
 function VSCode({ value, language, onChange }: Props) {
   const [focused, setFocused] = useState(false);
 
-  const { isDarkTheme } = useThemeContext();
+  const { colorMode, setColorMode } = useColorMode();
```

Comment:
It doesn't look like `setColorMode` is used
```suggestion
  const { colorMode } = useColorMode();
```